### PR TITLE
nats-ws

### PR DESCRIPTION
**Pull request by Capprin**
Adds a webservice to PDL that forwards message traffic on to clients.

To do:
- [x] Build webservice separately
- [x] Create PDL configuration that uses a NATS sender
- [ ] Run PDL in a container
- [x] Run WS (either locally or in a container)
- [x] Run NATS streaming container
- [x] Write html webage that connects to ws socket and streams notifications
- [x] Demo!

Demo available at [github.com/Capprin/pdl-demo](url)

### CODE REVIEW

**Comment by Capprin** [comment]
Finished WebSocketNotificationClient and constituent tests. Further commits will be fixes (if any) for end-to-end testing.

File: build.gradle [hunk 1]
```diff
@@ -117,6 +147,9 @@ javadoc {
   options.addBooleanOption('html5', true)
 }
 
+mainClassName = "gov.usgs.earthquake.distribution.Bootstrap"
+
+/*
```
Comment (jmfee-usgs) [inline build.gradle]: Remove the commented code if no longer needed.

File: nats-init [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```
Comment (jmfee-usgs) [inline nats-init]: move into etc , maybe etc/nats something

File: src/main/java/gov/usgs/earthquake/distribution/EIDSNotificationSender.java [hunk 3]
```diff
@@ -3,162 +3,40 @@
...
 	public static final String EIDS_DEFAULT_POLLDIR = "polldir";
 
-	/** The product storage where retrieved products are stored. */
-	private URLProductStorage productStorage;
-
-	/** How long to store retrieved product, in milliseconds. */
-	private Long productStorageMaxAge;
-
 	/** How long to wait until checking for expired notifications/products. */
 	private Long senderCleanupInterval;
```
Comment (jmfee-usgs) [inline src/main/java/gov/usgs/earthquake/distribution/EIDSNotificationSender.java]: I think this property is unused

File: src/main/java/gov/usgs/earthquake/nats/NatsStreamingConnectionTest.java [hunk 4]
```diff
@@ -0,0 +1,66 @@
+package gov.usgs.earthquake.nats;
```
Comment (jmfee-usgs) [inline src/main/java/gov/usgs/earthquake/nats/NatsStreamingConnectionTest.java]: move into src/test

File: nats-init (hunk 2 above)
Comment (Capprin) [reply nats-init]: named etc/nats_setup/nats-init

File: src/main/java/gov/usgs/earthquake/distribution/EIDSNotificationSender.java (hunk 3 above)
Comment (Capprin) [reply src/main/java/gov/usgs/earthquake/distribution/EIDSNotificationSender.java]: removed, along with getter/setter. No longer exists project-wide.

File: src/main/java/gov/usgs/earthquake/nats/NatsStreamingConnectionTest.java (hunk 4 above)
Comment (Capprin) [reply src/main/java/gov/usgs/earthquake/nats/NatsStreamingConnectionTest.java]: moved into src/test/nats

**Pull request closed without merging**